labop_convert/behavior_specialization.py

```diff
@@ -31,7 +31,7 @@ def __init__(self) -> None:
         self.execution = None
 
         # This data field holds the results of the specialization
-        self.data = None
+        self.data = []
```

Comment:
Is there a reason why this is a list?  I have been using this field to hold a singleton that the editor uses to render the specialization.

Reply:
There was a test failing somewhere because this was set to `None`.  If I remember correctly, I think you initialize the field by calling `on_begin`.  So in the other specializations,  I think I do the same, but I'm using the field as a list.  So at the moment it is arbitrary what it is initially set to, as long as it is not None -- it's actual type will get determined by `on_begin`.  This could probably all be sorted out better, but I needed a workaround.